R/pmin-method.R

```diff
@@ -0,0 +1,39 @@
+#'
+#' @title pmin/pmax
+#' @name magpie-pmin-pmax
+#' @aliases pmin pmax pmin,magpie-method pmin,ANY-method, pmax,magpie-method pmax,ANY-method
```

Comment:
Okay I'm a bit confused here, are there supposed to be two `pmin` in the beginning? Is there a comma missing? And two ANY-method? The following would be more intuitive to me:
```suggestion
#' @aliases pmin pmax,pmin,magpie-method pmin,ANY-method,pmax,magpie-method pmax
```

Reply:
Ah there it is! I find the roxygen + Rd syntax highly confusing in general o.O